### PR TITLE
uiux: ensure components have proper theme text color

### LINKED ISSUE #899: When in dark mode, there's some screens & modals that have text with a black color

### PR DESCRIPTION
**Pull request by PascalPrecht**
Closes #899

### CODE REVIEW

File: ui/app/AppLayouts/Profile/Sections/SyncContainer.qml [hunk 1]
```diff
@@ -23,9 +23,32 @@ Item {
...
+                radius: 13
+                border.color: control.down ? "#17a81a" : "#21be2b"
+
+                Rectangle {
+                    width: 14
+                    height: 14
+                    x: 6
+                    y: 6
+                    radius: 7
+                    color: control.down ? "#17a81a" : "#21be2b"
```
Comment (PascalPrecht) [inline ui/app/AppLayouts/Profile/Sections/SyncContainer.qml:43]: Can ignore these colors for now.

File: ui/shared/BlockContactConfirmationDialog.qml [hunk 2]
```diff
@@ -20,6 +20,7 @@ ModalPopup {
         anchors.left: parent.left
         anchors.right: parent.right
         wrapMode: Text.WordWrap
+        color: Style.current.textColor
```
Comment (jrainville) [inline ui/shared/BlockContactConfirmationDialog.qml:23]: No need to do this. Just change `Text` to `StyledText`. It handles the color and the font

File: ui/shared/StyledTextArea.qml [hunk 3]
```diff
@@ -32,7 +32,7 @@ Item {
         anchors.top: parent.top
         anchors.topMargin: 0
         font.pixelSize: 13
-        color: Style.current.black
+        color: Style.current.textColor
```
Comment (jrainville) [inline ui/shared/StyledTextArea.qml:35]: You could just remove the `color` since `StyledText` handles it

**Review by emizzle — APPROVED** [review]
Well done, I do agree with changing the `Text` components to `StyledText` to remove the color specification.

**Review by jrainville — APPROVED** [review]

**Pull request merged**